includes/Dashboard/Templates/Settings.php

```diff
@@ -524,6 +524,10 @@ function insert_settings_info() {
                 'phone'                    => sanitize_text_field( $post_data['setting_phone'] ),
                 'show_email'               => sanitize_text_field( $post_data['setting_show_email'] ),
                 'show_more_ptab'           => sanitize_text_field( $post_data['setting_show_more_ptab'] ),
+                'show_featured_products'   => ! empty( $post_data['setting_show_featured_products'] ) ? sanitize_text_field( $post_data['setting_show_featured_products'] ) : 'no',
```

Comment:
Isn't that `isset()` checking is enough for here?
We should use `isset()` and `empty()` in it's proper places based on scenario. Please check for other settings also.